src/overlays/actors/ovl_En_Mt_tag/z_en_mt_tag.c

```diff
@@ -154,7 +154,7 @@ s32 EnMttag_GetCurrentCheckpoint(Actor* actor, PlayState* play, s32* upcomingChe
 
     // Iterates through all possible checkpoints that are associated with this sceneExitIndex.
     do {
-        if ((Math3D_PointDistToLine2D(
+        if ((Math3D_PointDistSqToLine2DImpl(
```

Comment:
```suggestion
        if (Math3D_PointDistSqToLine2DImpl(
```
I believe there is an unnecessary set of parens here.